contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -55,6 +55,37 @@ webportal:
   server-port: 9286
 
 
+#If you want to customize the hivedscheduler vc config or add a new vc,
+#pls check https://github.com/microsoft/pai/blob/master/docs/hivedscheduler/devops.md
+hivedscheduler:
+  config: |
+    physicalCluster:
+      gpuTypes:
+        DT:
+          gpu: 1
+          cpu: {{ env["hived"]["unit-cpu"] }}
+          memory: {{ env["hived"]["unit-mem"] }}
+      cellTypes:
+        DT-NODE:
+          childCellType: DT
+          childCellNumber: {{ env["hived"]["min-allocatable-gpu"] }}
```

Comment:
remove "-allocatable" since unit- also does not have it.

Reply:
done